src/modules/jcms_rest/src/Plugin/rest/resource/CollectionListRestResource.php

```diff
@@ -55,7 +56,8 @@ public function get() {
         }
       }
     }
-    $response = new JsonResponse($response_data, Response::HTTP_OK, ['Content-Type' => 'application/vnd.elife.collection-list+json;version=1']);
+    $response = new JCMSRestResponse($response_data, Response::HTTP_OK, ['Content-Type' => $this->getContentType()]);
```

Comment:
two lines for each endpoint, making explicit the content type and the caching, good enough for me; I'll leave the rest to @nlisgo